pkg/env/env.go

```diff
@@ -62,6 +64,11 @@ func (e *Env) Environ() (vars []string) {
 	return vars
 }
 
+// Platform returns the platform of the dad executable
+func (e *Env) Platform() string {
+	return fmt.Sprintf("dad-%s-%s", runtime.GOOS, runtime.GOARCH)
+}
```

Comment:
This should be a package function, not a method on `Env`.

`// An Env provides a simple interface to manipulate environment variables`

Reply:
But even there, this string seems to be very specific to the upgrade mechanism.
Or maybe it deserves it's special package to contain more code about Dad itself. 

But since we don't have anything else for now, I would co-locate it to where it's used (helpers/upgrade)